src/components/index.js

```diff
@@ -91,6 +91,8 @@ export { default as FieldRadioButton } from './FieldRadioButton/FieldRadioButton
 export { default as FieldReviewRating } from './FieldReviewRating/FieldReviewRating';
 export { default as FieldSelect } from './FieldSelect/FieldSelect';
 export { default as FieldTextInput } from './FieldTextInput/FieldTextInput';
+export { default as CustomExtendedDataField } from './CustomExtendedDataField/CustomExtendedDataField';
```

Comment:
Move this line under the group title: _"// These components include other components //"_

It should not be defined before all of its child components are defined. The load order matters and might cause circular dependency issues. So, this is a more complex component than Field* components since it uses them internally.